app/src/main/res/values/strings.xml

```diff
@@ -478,13 +478,14 @@
     <string name="onboarding_incognito_title">Where incognito meets invisible</string>
 
     <!-- Onboarding: Incognito Description -->
-    <string name="onboarding_incognito_description">Focus is a privacy-first browser with extra-strong tracking and cookie protection built in.</string>
+    <!-- Text %1$s will be replaced with the name of the app (e.g. Focus, Klar) -->
+    <string name="onboarding_incognito_description">%1$s is a privacy-first browser with extra-strong tracking and cookie protection built in.</string>
```

Comment:
nit: If there is only string placeholder we could've used just `%s` without having to indicate that we want the first argument.